### PR TITLE
Add/schedule jobs

### PR DESCRIPTION
**Pull request by borismattijssen**

### CODE REVIEW

File: pkg/client/workflows_test.go [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	tpc, err := getClient(jsonList)
+	if err != nil {
+		t.Error("Error while creating client")
+	}
+	opts := k8sApi.ListOptions{}
+	list, err := tpc.Workflows(k8sApi.NamespaceAll).List(opts)
+	if err != nil {
+		t.Errorf("Error while listing workflows: %v", err)
+	}
+	if !reflect.DeepEqual(list, &expected) {
```
Comment (borismattijssen) [inline pkg/client/workflows_test.go]: This needs to be fixed.

File: pkg/workflow/manager.go [hunk 2]
```diff
@@ -95,7 +97,8 @@ func NewWorkflowManager(oldClient k8sClient.Interface, kubeClient clientset.Inte
 			AddFunc: wc.enqueueController,
 			UpdateFunc: func(old, cur interface{}) {
 				if workflow := cur.(*api.Workflow); !isWorkflowFinished(workflow) {
-					wc.enqueueController(workflow)
+					// wc.enqueueController(workflow)
```
Comment (borismattijssen) [inline pkg/workflow/manager.go]: We should find out why workflow is updated so frequently when it really isn't.

File: pkg/api/types.go [hunk 3]
```diff
@@ -37,7 +37,7 @@ type WorkflowSpec struct {
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
 	k8sApi.ObjectMeta `json:"metadata,omitempty"`
 
-	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,omitempty"`
+	ActiveDeadlineSeconds int64 `json:"activeDeadlineSeconds,omitempty"`
```
Comment (nov1n) [inline pkg/api/types.go]: Misschien hier toch een pointer aanhouden omdat het overal gedaan wordt?

File: pkg/client/workflows.go [hunk 4]
```diff
@@ -48,7 +48,11 @@ func newWorkflows(c *ThirdPartyClient, namespace string) *workflows {
 }
 
 func (w *workflows) List(opts k8sApi.ListOptions) (result *api.WorkflowList, err error) {
-	url := w.client.baseURL + "/namespaces/" + w.ns + "/workflows"
+	nsPath := "/namespaces/" + w.ns
```
Comment (nov1n) [inline pkg/client/workflows.go]: 51:54 evt vervangen door

```
var nsPath
if w.ns != "" {
    nsPath = "/namespaces/" + w.ns
}
```

File: pkg/workflow/manager.go [hunk 5]
```diff
@@ -163,6 +167,98 @@ func (w *WorkflowManager) worker() {
 }
 
 func (w *WorkflowManager) syncWorkflow(key string) error {
+	fmt.Println("Syncing: " + key)
+
+	startTime := time.Now()
+	defer func() {
+		glog.V(4).Infof("Finished syncing workflow %q (%v)", key, time.Now().Sub(startTime))
```
Comment (nov1n) [inline pkg/workflow/manager.go]: Choose between glog and system log

File: pkg/workflow/manager.go [hunk 6]
```diff
@@ -163,6 +167,98 @@ func (w *WorkflowManager) worker() {
...
+		glog.Infof("Waiting for job controller to sync, requeuing workflow %v", key)
+		w.queue.Add(key)
+		return nil
+	}
+
+	// Obtain the workflow object from store by key
+	obj, exists, err := w.workflowStore.Store.GetByKey(key)
+	if !exists {
+		glog.V(4).Infof("Workflow has been deleted: %v", key)
+		w.expectations.DeleteExpectations(key)
```
Comment (nov1n) [inline pkg/workflow/manager.go]: We still need to find out how our system uses expectations

File: pkg/api/types.go (hunk 3 above)
Comment (borismattijssen) [inline pkg/api/types.go]: Please use English ;)
I agree with you, but maybe we should fist ask Ad what the advantage of using pointers to integers is?

File: pkg/client/workflows.go (hunk 4 above)
Comment (borismattijssen) [inline pkg/client/workflows.go]: I think it should be this, but I agree. I'll fix. 
```nsPath := ""
if w.ns != "" {
    nsPath = "/namespaces/" + w.ns
}```

File: pkg/workflow/manager.go (hunk 5 above)
Comment (borismattijssen) [inline pkg/workflow/manager.go]: Will fix to use glog ;)

File: pkg/workflow/manager.go (hunk 6 above)
Comment (borismattijssen) [inline pkg/workflow/manager.go]: We should ask Dario ;)

**Pull request merged**